Entities/AHuman.cpp

```diff
@@ -3463,16 +3470,22 @@ void AHuman::Update()
             // Only go slower outward
 			if (m_SharpAimProgress < aimMag)
 				m_SharpAimProgress += (aimMag - m_SharpAimProgress) * 0.035;
-            else
-                m_SharpAimProgress = aimMag;
-        }
-        else
-            m_SharpAimProgress = 0;
+			else
+				m_SharpAimProgress = aimMag;
+			m_SharpAimRevertTimer.Reset();
+		}
+		else {
+			m_SharpAimProgress *= 0.95F;
```

Comment:
What it does is smoothly diminish sharp aim progress when the timer gets interrupted / reset somehow. It looks good ingame.

Would this be an adequate comment: `// Deplete sharp aim smoothly if the timer isn't done` or is it kind of pointless? Sounds pointless to me.

Reply:
Ahh, I guess before it jumped back to 0, and now it slides smoothly? I feel like this might be solveable with a simple LERP or EaseIn or whatever, instead of this timer business, but I might be wrong.

I agree that the comment isn't great, I'd say put a *changed entry in the changelog about it, so people know there's this new feature, even if it's minor.